cloud/ec2_fleet.go

```diff
@@ -343,36 +339,35 @@ func (m *ec2FleetManager) StartInstance(context.Context, *host.Host, string) err
 }
 
 func (m *ec2FleetManager) Cleanup(ctx context.Context) error {
-	if err := m.client.Create(m.credentials, m.region); err != nil {
+	if err := m.client.Create(ctx, m.credentials, m.region); err != nil {
 		return errors.Wrap(err, "creating client")
 	}
 	defer m.client.Close()
 
 	launchTemplates, err := m.client.GetLaunchTemplates(ctx, &ec2.DescribeLaunchTemplatesInput{
-		Filters: []*ec2.Filter{
-			{Name: aws.String("tag-key"), Values: []*string{aws.String(evergreen.TagDistro)}},
+		Filters: []types.Filter{
+			{Name: aws.String("tag-key"), Values: []string{evergreen.TagDistro}},
 		},
 	})
 	if err != nil {
 		return errors.Wrap(err, "getting launch templates")
 	}
 
 	catcher := grip.NewBasicCatcher()
-	deleted := []string{}
+	deletedCount := 0
 	for _, template := range launchTemplates {
 		if template.CreateTime != nil && template.CreateTime.Before(time.Now().Add(-launchTemplateExpiration)) {
 			_, err := m.client.DeleteLaunchTemplate(ctx, &ec2.DeleteLaunchTemplateInput{LaunchTemplateId: template.LaunchTemplateId})
 			catcher.Add(err)
 			if err == nil {
-				deleted = append(deleted, aws.StringValue(template.LaunchTemplateId))
+				deletedCount++
 			}
 		}
 	}
 
-	grip.InfoWhen(len(deleted) > 0, message.Fields{
+	grip.InfoWhen(deletedCount > 0, message.Fields{
 		"message":       "removed launch templates",
-		"deleted_count": len(deleted),
-		"deleted":       deleted,
+		"deleted_count": deletedCount,
```

Comment:
We don't want to capture the deleted template IDs anymore?

Reply:
It seemed the actual IDs aren't important.

For context, this is used in a cleanup job that gets any leftover templates so we don't have a bunch of leftover templates sitting around. So the count of how many templates are getting left over is potentially interesting (like [this query](https://mongodb.splunkcloud.com/en-US/app/search/search?q=search%20index%3Devergreen%20%22removed%20launch%20templates%22%0A%7C%20bin%20_time%20span%3D1d%0A%7C%20stats%20sum(deleted_count)%20by%20_time&display.page.search.mode=verbose&dispatch.sample_ratio=1&workload_pool=standard_perf&earliest=-7d%40h&latest=now&display.page.search.tab=statistics&display.general.type=statistics&sid=1685994965.4950198)), but the actual IDs don't really matter.